scene/resources/resource_format_text.cpp

```diff
@@ -1495,6 +1551,24 @@ ResourceUID::ID ResourceFormatLoaderText::get_resource_uid(const String &p_path)
 	return loader.get_uid(f);
 }
 
+String ResourceFormatLoaderText::get_attached_script_path(const String &p_path) const {
+	String type = ResourceLoader::get_resource_type(p_path);
+
+	// We assume that only Resources can have attached scripts (excluding Scripts, which still extend Resource).
+	if (ClassDB::is_parent_class(type, Script::get_class_static()) || !ClassDB::is_parent_class(type, Resource::get_class_static())) {
```

Comment:
Again, the condition can be swapped.

Reply:
Whoops I was supposed to use the helper method here